client/starwhale/core/model/model.py

```diff
@@ -268,13 +269,19 @@ def eval_user_handler(
             else:
                 _scheduler.schedule_single_task(step, task_index)
         except Exception as e:
-            _manifest["status"] = STATUS.FAILED
+            _status = STATUS.FAILED
```

Comment:
`_status=` in line 256 and line 272 is useless.
`_status` in finally code block will overwrite it.

Reply:
yes, already change to try block